### PR TITLE
add html question

### PR DESCRIPTION
**Pull request by arunsridher**

### CODE REVIEW

File: html/answers/arunsridher-q1.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        gtag('config', 'UA-157837769-1');
+        </script>
+        
+        <title>
+            HTML | InterQues | Coding Ninjas
+        </title>
+
+        <link rel="shortcut icon" type="image/png" href="../../assets/favicon.png"/>
+
+        <meta name="description" content="Why do we need to put DOCTYPE tag at the begining of every HTML file?">
```
Comment (shivamyth) [inline html/answers/arunsridher-q1.html]: wrong content in meta tag

**Review by shivamyth — DISMISSED** [review]
There is an unnecessary submodule folder remove it and do the changes as requested.

File: html/answers/arunsridher-q1.html (hunk 1 above)
Comment (arunsridher) [reply html/answers/arunsridher-q1.html]: what should be the value of the content??

File: html/answers/arunsridher-q1.html [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        gtag('config', 'UA-157837769-1');
+        </script>
+        
+        <title>
+            HTML | InterQues | Coding Ninjas
+        </title>
+
+        <link rel="shortcut icon" type="image/png" href="../../assets/favicon.png"/>
+
+        <meta name="description" content="HTML interview questions">
```
Comment (dakshkhetan) [inline html/answers/arunsridher-q1.html]: Change content="" to question title.

**Review by dakshkhetan — CHANGES_REQUESTED** [review]
Delete/remove the unnecessary sub folder

**Review by dakshkhetan — APPROVED** [review]

**Pull request merged**